workflows/spaceranger/spaceranger_count.wdl

```diff
@@ -232,8 +232,15 @@ task run_spaceranger_count {
             return darkimages
 
         has_cyta = not_null('~{cytaimage}')
+        probe_set = "~{probe_set}"
```

Comment:
Why do we need Line 235 and 236?

Reply:
Less side effects

Reply:
what is wrong with defining a variable instead? not sure why this is imperative